examples/ExportUsersToCSV/exportUsersToCSV.go

```diff
@@ -61,8 +95,29 @@ func main() {
 		out(fmt.Sprintf("%t", user.PasswordExpired))
 		out(fmt.Sprintf("%t", user.Sudo))
 
-		for _, tag := range user.Tags {
-			out(tag.Name)
+		if isGroups {
+			// For now, just list the User Groups this user is a member of.
+			// NOTE: there are many more associations for a user in a Groups org we may want to list here as well:
+			// Applications, Directories, GSuite, LDAP, O365, Systems, Radius Servers
+
+			var graphs []jcapiv2.GraphObjectWithPaths
```

Comment:
Same as above with the loop breakout stuff, though this is simple enough that you could probably just get away with the `continue` after your `if err != nil` check and still have it be pretty readable.